### PR TITLE
docs(docs-site): added documentation for PORTERS

### PR DESCRIPTION
**Pull request by scermat**
Added documentation with reference to the PORTERS service and the Taiko Name Service API provided by PORTERS.

This was done in line with the ask requested from @benedictvscriticus.

## Sections Updated
- Resources / Developer Tools: Added card with a link to PORTERS
- Resources: Added node providers page with a high level overview of what PORTERS provides in relation to Taiko.

### CODE REVIEW

**Comment by RogerLamTd** [comment]
@scermat I believe this may belong in our ecosystem page, not our documentation. Please feel free to open a issue [here](https://github.com/taikoxyz/website-v2/issues)

**Comment by benedictvscriticus** [comment]
Hi @RogerLamTd! Thank you for your feedback, however @swarna1101 specifically requested us to add this to your documentation. Please clarify this matter internally and inform us over our established channels with the Taiko team where you would like us to add this.

If you prefer to sort this out outside of a PR, we are also happy to provide you with the necessary information for you to add this wherever you see fit.

Thank you for your understanding.

**Comment by RogerLamTd** [comment]
> Hi @RogerLamTd! Thank you for your feedback, however @swarna1101 specifically requested us to add this to your documentation. Please clarify this matter internally and inform us over our established channels with the Taiko team where you would like us to add this.
> 
> If you prefer to sort this out outside of a PR, we are also happy to provide you with the necessary information for you to add this wherever you see fit.
> 
> Thank you for your understanding.

Ah I wasn't aware! Let me confirm and get back to you.

**Comment by benedictvscriticus** [comment]
> > Hi @RogerLamTd! Thank you for your feedback, however @swarna1101 specifically requested us to add this to your documentation. Please clarify this matter internally and inform us over our established channels with the Taiko team where you would like us to add this.
> > If you prefer to sort this out outside of a PR, we are also happy to provide you with the necessary information for you to add this wherever you see fit.
> > Thank you for your understanding.
> 
> Ah I wasn't aware! Let me confirm and get back to you.

No worries, just let us know and we'll take it from there.

**Comment by RogerLamTd** [comment]
@benedictvscriticus Checked with @swarna1101, we can add to docs!

File: packages/docs-site/src/content/docs/resources/node-providers.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (RogerLamTd) [inline packages/docs-site/src/content/docs/resources/node-providers.md:1]: I don't think we will keep a page specifically for node providers.

File: packages/docs-site/src/content/docs/resources/developer-tools.mdx [hunk 2]
```diff
@@ -38,4 +38,14 @@ import { LinkCard, CardGrid } from "@astrojs/starlight/components";
     description="API3 offers over 200 first-party, OEV-enabled price feeds on Taiko Mainnet and Testnet, providing reliable and decentralized data through the API3 Market."
     href="https://market.api3.org/taiko"
   />
+</CardGrid>
+
+## APIs
```
Comment (RogerLamTd) [inline packages/docs-site/src/content/docs/resources/developer-tools.mdx:43]:
```suggestion
## APIs
## Node Providers
```

**Review by scermat — COMMENTED** [review]
From my side happy for you to proceed internally, feel free to apply any tweaks you deem appropriate to fit within the wider Taiko docs

**Review by RogerLamTd — APPROVED** [review]

**Review by swarna1101 — APPROVED** [review]

**Pull request merged**